docs/content/installation/configuration.md

```diff
@@ -55,7 +55,7 @@ Operator.
 | `pgo_add_os_ca_store` | false | **Required** | When true, includes system default certificate authorities. |
 | `pgo_admin_password` | examplepassword | **Required** | Configures the pgo administrator password. |
 | `pgo_admin_perms` | * | **Required** | Sets the access control rules provided by the PostgreSQL Operator RBAC resources for the PostgreSQL Operator administrative account that is created by this installer. Defaults to allowing all of the permissions, which is represented with the * |
-| `pgo_admin_role_name` | pgoadmin | **Required** | Sets the name of the PostgreSQL Operator role that is utilized for administrative operations performed by the PostgreSQL Operator. |
+| `pgo_admin_role_name` | pgoadmin |  | Sets the name of the PostgreSQL Operator role that is utilized for administrative operations performed by the PostgreSQL Operator. If not set, a randomly generated password will be set.|
```

Comment:
`pgo_admin_password` was removed from the preflight check. I suspect this is a typo intended for L56?

Reply:
yep, that's correct. Thanks for catching that. Updated.